magenta/interfaces/midi/magenta_midi.py

```diff
@@ -89,7 +89,9 @@
 tf.app.flags.DEFINE_integer(
     'temperature_control_number',
     None,
-    'The control change number to use for controlling softmax temperature.')
+    'The control change number to use for controlling softmax temperature.'
+    'This should be an integer between 0 and 127 corresponding to a softmax '
+    'temperature between 0.1 and 2.')
```

Comment:
I agree that it's good to include the range. However, I want to make it clear that the flag value doesn't directly determine the temperature. Instead it specifies which control change to monitor whose value will be mapped to the range. How about:

The control change number to use for controlling softmax temperature. The value of control changes with this number will be used to set the temperature in a linear range between 0.1 and 2.